Elevel/Elevel.Api/Startup.cs

```diff
@@ -32,6 +32,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddAutoMapper(typeof(TestQuestionProfile));
+            services.AddAutoMapper(typeof(Audition));
```

Comment:
you should add only MapperProfiles, and 1 will include full assembly, you don't need this